build_locally.sh

```diff
@@ -45,8 +45,10 @@ if ! have_noinstall "$@"; then
 fi
 
 # Test website using same script as Travis
-if have_travis "$@"; then
-  ./.travis.sh
+if have_github_actions "$@"; then
+  export NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+  bundle exec jekyll build --strict_front_matter
+  bundle exec htmlproofer ./_site --check-html --only-4xx --http-status-ignore 429 --alt-ignore '/.*/'
```

Comment:
Sorry, I didn't remember that these commands are used here as well.
To avoid code duplication, I suggest keeping the shell file.